plugins/operators/mattermost_operator.py

```diff
@@ -0,0 +1,30 @@
+from airflow.utils.decorators import apply_defaults
+import requests
+
+class MattermostWebhookOperator():
+
+    @apply_defaults
+    def __init__(self,
+                 http_conn_url=None,
+                 message=None,
+                 channel=None,
+                 username=None,
+                 **kwargs) -> None:
+        self.http_conn_url = http_conn_url
+        self.message = message
+        self.channel = channel
+        self.username = username
+
+    def execute(self, context): 
+        print("executing post in mattermost operator- ", self.http_conn_url)
```

Comment:
```suggestion
        self.log.info("executing post in mattermost operator")
```

A better alternative for `print` is airflow's native logging. In general it's better to rely on python's logging rather than print function. It 

`http_conn_url` contains the webhook URL which can be considered as sensitive information. Logging/printing sensitive information must be avoided.